api/client/lib.py

```diff
@@ -438,7 +438,11 @@ def get_data_series(access_token, api_host, **selection):
     params = get_params_from_selection(**selection)
     resp = get_data(url, headers, params)
     try:
-        return resp.json()['data']
+        response = resp.json()['data']
+        if any(map(lambda series: 
+            series['historical_region'] or series['historical_partner_region'], response)):
```

Comment:
Use `series.get('historical_region', False) or series.get('historical_partner_region', False)` instead of square brackets so we can make this field optional.

Reply:
done

Reply:
Should be False, not True

Reply:
Also the unit test here https://github.com/gro-intelligence/api-client/blob/fbba757170e47c3075490b8608e26ee3fffef2af/api/client/lib_test.py#L11 has these responses mocked out as strings instead of as dicts.

Need to update that to match the expected format

Reply:
> Should be False, not True

should be True. if series.get(includes_historical_region, True) warn users ?